rai/node/rpc.cpp

```diff
@@ -4228,23 +4228,40 @@ void rai::rpc_handler::work_generate ()
 	if (rpc.config.enable_control)
 	{
 		std::string hash_text (request.get<std::string> ("hash"));
+	    bool use_peers (request.get_optional<bool> ("use_peers") == true);
```

Comment:
I think the preferred way to do this is set the default value to false.

Reply:
You are maybe right. It's always done like this with optional bools though. So convention seems like this.